packages/react/index.modern.fb.js

```diff
@@ -33,6 +33,7 @@ export {
   unstable_Activity,
   unstable_Scope,
   unstable_SuspenseList,
+  unstable_TracingMarker,
```

Comment:
this was missing from the modern entrypoint but is necessary for tracing tests to pass. Curious though that it doesn't fail. probably because it's a symbol and it imports in cjs as undefined